src/frontends/android/app/src/main/jni/libandroidbridge/backend/android_service.c

```diff
@@ -157,6 +157,7 @@ static job_requeue_t handle_plain(private_android_service_t *this)
     struct pollfd pfd_read;
     int timeout = 1000;
     pfd_read.fd = tunfd;
+    pfd_read.events = POLLIN;
```

Comment:
- Does it work?
- Only POLLIN is required? How about POLLLOUT?

Reply:
works with POLLIN

Reply:
It is only to read info, the same logic was with select